setup.py

```diff
@@ -52,6 +52,6 @@ def read(filename):
     packages=PEP420PackageFinder.find(include=["lumicks.*"]),
     python_requires='>=3.6',
     install_requires=['pytest>=3.5, <4.0', 'h5py>=2.9, <3.0', 'numpy>=1.14, <2',
```

Comment:
I think that we can drop the `pytest` upper bound as well. It's up to v5 at this point and lots of packages use it.

Reply:
👍